incubator/solr/requirements.yaml

```diff
@@ -0,0 +1,6 @@
+---
+
+dependencies:
+  - name: zookeeper
+    version: 1.2.2
```

Comment:
This version can be a range. The patterns you can use can be found at https://github.com/masterminds/semver. The lock file will pin to the version when the dependencies are updated.